src/settings/ActionProfiles.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import { Pane } from '@folio/stripes/components';
+
+export default class ActionProfiles extends Component {
```

Comment:
Would it be better to just create another component or HOC to get rid of code reuse in such components as ActionProfiles, FieldMappingProfiles, FileExtensions and others? Seems like all of those components are having the same code except FormattedMessage. Please correct me if I am wrong.
And also maybe it would be better to use SFC (functional component) as there is no hooks and state in these components.

Reply:
Discussed it. Leave as is for now.